cmd/cli/cmd/switch/switch.go

```diff
@@ -0,0 +1,65 @@
+package datumswitch
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	datum "github.com/datumforge/datum/cmd/cli/cmd"
+	"github.com/datumforge/datum/internal/httpserve/handlers"
+	"github.com/datumforge/datum/pkg/datumclient"
+)
+
+var loginCmd = &cobra.Command{
+	Use:   "switch",
+	Short: "switch organization contexts",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := switchorg(cmd.Context()); err != nil {
+			return err
+
+		}
+		return nil
```

Comment:
you can simplify this 
```suggestion
		return switchorg(cmd.Context())
```